.travis.yml

```diff
@@ -4,7 +4,7 @@ os:
 
 sudo: false
 
-dist: trusty
+dist: xenial
```

Comment:
I had a look at the travis documentation, and apparently bionic is now supported as well - should we move straight to bionic?

Reply:
bionic is LTS, lets give it a go, at worst we just need to revert.